django/db/migrations/operations/models.py

```diff
@@ -277,70 +278,50 @@ def deconstruct(self):
             kwargs
         )
 
+    def _get_model_key(self, remote_model, app_label, model_name):
```

Comment:
perhaps `_get_model_tuple` to parallel naming of `db.models.utils.make_model_tuple()`?
`remote_model` -> `remote_model_name`?